src/Fantomas.Core.Tests/IfThenElseTests.fs

```diff
@@ -2648,3 +2648,28 @@ else
 
 printfn "d"
 """
+
+[<Test>]
+let ``duplicate newline after shifting trivia between else if, 2752`` () =
```

Comment:
Please use the original code with the reported config for the test with the number in it. It is ok to have another test like this, but you want to have a regression test exactly as the user had it in their code.